src/zero/service/postgres.clj

```diff
@@ -93,6 +95,61 @@
                          (util/getenv "USER" "postgres")
                          "password")))
 
+(defn get-table
+  "Return TABLE using transaction TX."
+  [tx table]
+  (jdbc/query tx (format "SELECT * FROM %s" table)))
```

Comment:
Does jdbc provides built-in protection against SQL injections? Cannot find any useful info https://github.com/clojure/java.jdbc

Reply:
I don't see a path to injection, but we could use prepared statements.